### ISSUE TITLE
how to set connect_timeout_ms outside of Bootstrap？

### ISSUE THREAD

**Issue by wenweihu86** [netty/netty#8696]
Expected behavior
since connect_timeout_ms is set in the Bootstrap, so different cluster server should have different Bootstrap instance. But Bootstrap instantiation is time consuming.
could you please upgrade netty to make connect_timeout_ms to be set out side of Bootstrap？
so that I can make Bootstrap single instance.

Actual behavior
if a module has to connect to two cluster servers, and has different connect timeout for different cluster,
I can only create two bootstrap since connect_timeout is set on Bootstrap.
creating Bootstrap is time consuming, so I want to create single instance of Bootstrap for all servers.

Steps to reproduce
no

Minimal yet complete reproducer code (or URL to code)
no

Netty version
4.1.30.Final

JVM version (e.g. java -version)
OS version (e.g. uname -a)

**Comment by normanmaurer** [comment]
Creating a Bootstrap is cheap... why you think it’s time consuming?

**Comment by wenweihu86** [comment]
Creating a Bootstrap need to create io threads, this is time consuming on real time.
If we can creating single bootstrap instance at start time, it will be more faster.
To do this, it just need to add connect_timeout_ms parameter for Bootstrap.connect function.

**Comment by normanmaurer** [comment]
That’s not correct you can reuse the same EventLoopGroup across different Bootstrap instances. That’s one of the reasons why it’s cheap to create new instances
